addons/hr_holidays/models/hr_leave_accrual_plan_level.py

```diff
@@ -70,8 +81,14 @@ class AccrualPlanLevel(models.Model):
         ('sun', 'Sunday'),
     ], default='mon', required=True, string="Allocation on")
     first_day = fields.Integer(default=1)
+    first_day_display = fields.Selection(
+        DAY_SELECT, compute='_compute_days_display', inverse='_inverse_first_day_display', required=True)
```

Comment:
Strange to have a compute / inverse with a required. Indeed this should be managed in field compute method that should always return a value. If you want to avoid people voiding it, required should be set in the view but not at model level (probably valid for other new fields).

Reply:
It is indeed only for the views to avoid empty values, I will move those to the views definitions instead